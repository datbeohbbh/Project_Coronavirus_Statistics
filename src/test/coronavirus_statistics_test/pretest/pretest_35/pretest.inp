50 100
q 1 2 97
q 1 17 459
? 7
q 1 43 159
q 2 43 108
? 4 3
q 1 16 438
q 1 42 197
? 7
q 1 21 207
? 4 6
q 1 49 61
? 4 6
q 1 1 96
? 4 2
q 1 18 288
q 1 25 270
q 2 21 73
q 1 40 362
? 4 3
q 1 6 122
q 1 37 129
q 1 35 330
? 4 9
q 1 31 90
q 1 45 190
q 2 40 187
q 2 6 116
q 2 18 166
q 2 31 42
q 2 43 1
q 1 30 317
q 1 11 58
? 4 4
q 2 43 18
? 4 6
q 2 6 5
? 6
q 1 39 76
q 3 43 27
q 3 42 94
q 2 18 62
q 3 45 168
? 6
? 4 12
q 1 32 391
? 4 1
q 3 1 58
q 3 11 36
q 3 43 2
? 4 11
q 3 25 78
q 2 1 4
q 1 23 304
? 6
q 1 27 296
q 1 46 100
? 6
q 2 31 30
q 2 23 137
? 4 21
q 2 43 3
? 6
q 1 50 432
q 2 40 18
q 2 35 85
? 4 2
? 4 2
? 7
? 4 11
q 3 46 88
? 6
q 2 30 303
q 3 16 226
q 3 37 83
? 4 5
q 3 1 22
? 4 14
? 7
? 4 8
? 7
? 6
? 6
? 4 22
? 7
? 6
? 7
? 4 6
? 6
? 4 9
? 4 13
? 6
? 4 7
? 4 10
? 6
? 4 20
? 6
? 7
? 7
? 7