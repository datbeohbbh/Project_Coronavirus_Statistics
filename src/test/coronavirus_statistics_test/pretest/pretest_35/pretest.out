Query ? 7 :
Currently, no deaths have been recorded

Query ? 4 3 :
The answer is the city with the index 17 
- Information of the city 17 -
-> Number of current patients : 459
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
Currently, no deaths have been recorded

Query ? 4 6 :
The answer is the city with the index 17 
- Information of the city 17 -
-> Number of current patients : 459
-> Has recovered : 0
-> Was dead : 0

Query ? 4 6 :
The answer is the city with the index 16 
- Information of the city 16 -
-> Number of current patients : 438
-> Has recovered : 0
-> Was dead : 0

Query ? 4 2 :
The answer is the city with the index 49 
- Information of the city 49 -
-> Number of current patients : 61
-> Has recovered : 0
-> Was dead : 0

Query ? 4 3 :
The answer is the city with the index 1 
- Information of the city 1 -
-> Number of current patients : 96
-> Has recovered : 0
-> Was dead : 0

Query ? 4 9 :
The answer is the city with the index 25 
- Information of the city 25 -
-> Number of current patients : 270
-> Has recovered : 0
-> Was dead : 0

Query ? 4 4 :
The answer is the city with the index 11 
- Information of the city 11 -
-> Number of current patients : 58
-> Has recovered : 0
-> Was dead : 0

Query ? 4 6 :
The answer is the city with the index 1 
- Information of the city 1 -
-> Number of current patients : 96
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 40
Quantity: 187 people

Query ? 6 :
The city with the highest recovery is city with the index 18
Quantity: 228 people

Query ? 4 12 :
The answer is the city with the index 37 
- Information of the city 37 -
-> Number of current patients : 129
-> Has recovered : 0
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 6 
- Information of the city 6 -
-> Number of current patients : 1
-> Has recovered : 121
-> Was dead : 0

Query ? 4 11 :
The answer is the city with the index 42 
- Information of the city 42 -
-> Number of current patients : 103
-> Has recovered : 0
-> Was dead : 94

Query ? 6 :
The city with the highest recovery is city with the index 18
Quantity: 228 people

Query ? 6 :
The city with the highest recovery is city with the index 18
Quantity: 228 people

Query ? 4 21 :
The answer is the city with the index 32 
- Information of the city 32 -
-> Number of current patients : 391
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 18
Quantity: 228 people

Query ? 4 2 :
The answer is the city with the index 31 
- Information of the city 31 -
-> Number of current patients : 18
-> Has recovered : 72
-> Was dead : 0

Query ? 4 2 :
The answer is the city with the index 31 
- Information of the city 31 -
-> Number of current patients : 18
-> Has recovered : 72
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 45
Quantity: 168 people

Query ? 4 11 :
The answer is the city with the index 42 
- Information of the city 42 -
-> Number of current patients : 103
-> Has recovered : 0
-> Was dead : 94

Query ? 6 :
The city with the highest recovery is city with the index 18
Quantity: 228 people

Query ? 4 5 :
The answer is the city with the index 11 
- Information of the city 11 -
-> Number of current patients : 22
-> Has recovered : 0
-> Was dead : 36

Query ? 4 14 :
The answer is the city with the index 21 
- Information of the city 21 -
-> Number of current patients : 134
-> Has recovered : 73
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 16
Quantity: 226 people

Query ? 4 8 :
The answer is the city with the index 37 
- Information of the city 37 -
-> Number of current patients : 46
-> Has recovered : 0
-> Was dead : 83

Query ? 7 :
The city with the highest death is city with the index 16
Quantity: 226 people

Query ? 6 :
The city with the highest recovery is city with the index 30
Quantity: 303 people

Query ? 6 :
The city with the highest recovery is city with the index 30
Quantity: 303 people

Query ? 4 22 :
The answer is the city with the index 50 
- Information of the city 50 -
-> Number of current patients : 432
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 16
Quantity: 226 people

Query ? 6 :
The city with the highest recovery is city with the index 30
Quantity: 303 people

Query ? 7 :
The city with the highest death is city with the index 16
Quantity: 226 people

Query ? 4 6 :
The answer is the city with the index 11 
- Information of the city 11 -
-> Number of current patients : 22
-> Has recovered : 0
-> Was dead : 36

Query ? 6 :
The city with the highest recovery is city with the index 30
Quantity: 303 people

Query ? 4 9 :
The answer is the city with the index 18 
- Information of the city 18 -
-> Number of current patients : 60
-> Has recovered : 228
-> Was dead : 0

Query ? 4 13 :
The answer is the city with the index 42 
- Information of the city 42 -
-> Number of current patients : 103
-> Has recovered : 0
-> Was dead : 94

Query ? 6 :
The city with the highest recovery is city with the index 30
Quantity: 303 people

Query ? 4 7 :
The answer is the city with the index 45 
- Information of the city 45 -
-> Number of current patients : 22
-> Has recovered : 0
-> Was dead : 168

Query ? 4 10 :
The answer is the city with the index 49 
- Information of the city 49 -
-> Number of current patients : 61
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 30
Quantity: 303 people

Query ? 4 20 :
The answer is the city with the index 27 
- Information of the city 27 -
-> Number of current patients : 296
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 30
Quantity: 303 people

Query ? 7 :
The city with the highest death is city with the index 16
Quantity: 226 people

Query ? 7 :
The city with the highest death is city with the index 16
Quantity: 226 people

Query ? 7 :
The city with the highest death is city with the index 16
Quantity: 226 people

